myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java

```diff
@@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
       //something is broken
       throw new RuntimeException("Could not find valid updateFile Method");
     }
+    
+    startInternal();
```

Comment:
Cool, will update and checkin after 21 Pilots concert! :)

Sent from my iPhone

> On Jun 10, 2016, at 7:39 PM, Darin <notifications@github.com> wrote:
> 
> In myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java:
> 
> > @@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
> >        //something is broken
> >        throw new RuntimeException("Could not find valid updateFile Method");
> >      }
> > +    
> > +    startInternal();
> VisibleForTesting is fine. Just concerned about the extra call to startInternal.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>